src/index.css

```diff
@@ -27,6 +31,13 @@ body{
 html, body, #root, #app, .full-height{
   height: 100%;
 }
+
+.flex-container{
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+}
+
 .full-height{
   height: calc(100% - 445px); /* not really a good way to do it, will probably change to flex column layout */
```

Comment:
Not in this PR but I'd reeeeeally suggest avoiding multi-unit calculations like this

Reply:
Something like `height: 95%` should work?

Reply:
Oh, for sure. We don't actually need this styling at all, it's just there to remind me to continue thinking of a better way to have each content section take up the entire page for the future (similar to the old layout, but actually done right w/ flex)